test/functional/Java11andUp/playlist.xml

```diff
@@ -138,4 +138,35 @@
 		</subsets>
 	</test>
 
+
+	<!--
+		Following test covers garbage collection tests scenario for object, primitive and exception constant dynamics.
+	-->
+	<test>
+		<testCaseName>CondyGarbageCollection</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+			<variation>-Xgcpolicy:optthruput</variation>
+			<variation>-Xgcpolicy:gencon</variation>
+			<variation>-Xgcpolicy:balanced</variation>
+			<variation>-Xgcpolicy:metronome</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+			-cp $(Q)$(LIB_DIR)$(D)asm-7.0.jar$(P)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
+			org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) -testnames GarbageCollectionCondyTest \
+			-groups $(TEST_GROUP) \
+			-excludegroups $(DEFAULT_EXCLUDE); \
+			$(TEST_STATUS)
+		</command>
+
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+			<subset>SE110</subset>
```

Comment:
this should be 11.

Reply:
There were recent changes made by Renfei regarding this. Rebase with latest to double check that it still works

Reply:
Works with 11